nest/src/progressbar.rs

```diff
@@ -39,27 +39,32 @@ impl ProgressBar {
     /// colors, and cannot go over 8 chars.
     /// Maximum value is 100.
     pub fn new(action: String) -> ProgressBar {
+        let refresh_rate = Duration::new(0, NANOS_PER_SEC / 10);
         ProgressBar {
             current: 0,
             max: 100,
             action,
             target: String::new(),
-            is_finished: false,
             start_time: Instant::now(),
-            last_time: Instant::now(),
+            last_time: Instant::now() - refresh_rate, // Make sure the progress bar will be drawed on first update
```

Comment:
Could have a variable `next_time` instead. (I think it may simplify logic below too).

Reply:
You know what's funny? I always use this method too, it's just the fisrt time I stop and think about it X)